Sources/StreamChatUI/ChatChannelList/ChatChannelListVC.swift

```diff
@@ -195,7 +195,7 @@ open class _ChatChannelListVC<ExtraData: ExtraDataTypes>: _ViewController,
         Animate { self.collectionView.layoutIfNeeded() }
     }
 
-    public func swipeableViewActionViews(for indexPath: IndexPath) -> [UIView] {
+    open func swipeableViewActionViews(for indexPath: IndexPath) -> [UIView] {
         let deleteView = CellActionView().withoutAutoresizingMaskConstraints
```

Comment:
Shouldn't this CellActionView() come from the components? 🤔

Reply:
don't blame the messenger 😂

Reply:
Ahah. Maybe @DominikBucher12 knows about this? 👀

Reply:
The `CellActionView` is not audited yet, but you are right, it should come from components...